### PR TITLE
Refactoring (users page) and style fix

### PR DESCRIPTION
**Pull request by KateVuilova**

### CODE REVIEW

File: src/containers/UsersPage/UsersPage.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ class UsersPage extends Component {
                 {
                     selectedUser: `${this.props.usersData[itemId].firstName} ${
                         this.props.usersData[itemId].lastName
-                    }`,
+                        }`,
```
Comment (mrmckeb) [inline src/containers/UsersPage/UsersPage.js]: This looks a little strange, was it prettier?

File: src/containers/UsersPage/UsersPage.js [hunk 2]
```diff
@@ -148,32 +150,29 @@ class UsersPage extends Component {
                                 this.questionModalDialog = dialog;
                             }}
                             headerText={
-                                !isAdmin ||
-                                (isAdmin && this.props.userData._id === this.state.selectedUserId)
+                                !isAdmin || isAdminWithIdMatch
```
Comment (mrmckeb) [inline src/containers/UsersPage/UsersPage.js]: How about:
```
const isLeaveAccount = !isAdmin || isAdminWithIdMatch;

...

isLeaveAccount ? "LEAVE" : "REMOVE";
```
As that logic also appears a few times.

Comment (mrmckeb) [reply src/containers/UsersPage/UsersPage.js]: You can name it whatever you like though.

File: src/containers/UsersPage/UsersPage.js (hunk 1 above)
Comment (KateVuilova) [reply src/containers/UsersPage/UsersPage.js]: It was problem with extensions in my environment. Fixed

File: src/components/List/List.js [hunk 3]
```diff
@@ -36,16 +36,15 @@ class List extends Component {
...
                 key={key}
-                className={
-                    column.className
-                        ? value[column.invalidParam]
-                            ? `${column.className} ${column.invalidClassName}`
-                            : column.className
-                        : ""
-                }
+                className={column.className
+                        ? checkedInputClassname : ""}
```
Comment (vdanyliv) [inline src/components/List/List.js]: maybe we can use this statement `checkedInputClassname || ""`

**Pull request merged**